### PR TITLE
feat: [SIG-584]: moved facing issue btn tracking from trackEvent to logEvent

### PR DESCRIPTION
**Pull request by SagarRajput-7**
### Summary

Moved facing issue btn tracking from trackEvent to logEvent

#### Screenshots

https://github.com/SigNoz/signoz/assets/162284829/9d56b41d-0a00-47b2-9434-5038b9f4f28c

### CODE REVIEW

**Review by makeavish — DISMISSED** [review]

File: frontend/src/hooks/analytics/useAnalytics.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import logEvent from 'api/common/logEvent';
```
Comment (Vikrant2520) [inline frontend/src/hooks/analytics/useAnalytics.tsx:1]: why are we importing `logEvent` in the file and exporting it from the hook ? can we directly use it ?

File: frontend/src/container/OnboardingContainer/common/ModuleStepsContainer/ModuleStepsContainer.tsx [hunk 2]
```diff
@@ -381,11 +381,12 @@ export default function ModuleStepsContainer({
 	};
 
 	const handleFacingIssuesClick = (): void => {
-		trackEvent('Onboarding V2: Facing Issues Sending Data to SigNoz', {
+		logEvent('Onboarding V2: Facing Issues Sending Data to SigNoz', {
```
Comment (Vikrant2520) [inline frontend/src/container/OnboardingContainer/common/ModuleStepsContainer/ModuleStepsContainer.tsx:384]: NIT: we do not need any error / success handling here ? as in we are not concerned with API returns whatever response here ?

Comment (SagarRajput-7) [reply frontend/src/container/OnboardingContainer/common/ModuleStepsContainer/ModuleStepsContainer.tsx:384]: yes, here we are not concerned about the response and also dont want to block the process due to this call.

File: frontend/src/hooks/analytics/useAnalytics.tsx (hunk 1 above)
Comment (SagarRajput-7) [reply frontend/src/hooks/analytics/useAnalytics.tsx:1]: Majorly done just to categorize this logEvent under useAnalytics, as its a similar option as trackEvent

File: frontend/src/container/OnboardingContainer/common/ModuleStepsContainer/ModuleStepsContainer.tsx (hunk 2 above)
Comment (Vikrant2520) [reply frontend/src/container/OnboardingContainer/common/ModuleStepsContainer/ModuleStepsContainer.tsx:384]: so if somebody wants to use the `logEvent` they will have to process the entire hook code. We should keep it separate.

File: frontend/src/hooks/analytics/useAnalytics.tsx (hunk 1 above)
Comment (Vikrant2520) [reply frontend/src/hooks/analytics/useAnalytics.tsx:1]: so if somebody wants to use the logEvent they will have to process the entire hook code. We should keep it separate.

Comment (SagarRajput-7) [reply frontend/src/hooks/analytics/useAnalytics.tsx:1]: Sure, will use logEvent directly here

**Review by Vikrant2520 — APPROVED** [review]

**Review by YounixM — APPROVED** [review]

**Pull request merged**